gulp-tasks/svg-sprite.js

```diff
@@ -27,28 +28,28 @@ const config = {
   },
 };
 
-gulp.task("build-sprite", (done) => {
+function buildSprite(done) {
   return (
-    gulp
-      .src(`${svgPath}/usa-icons/**/*.svg`)
+    src(`${svgPath}/usa-icons/*.svg`)
       .pipe(svgSprite(config))
-      // eslint-disable-next-line no-console
-      .on("error", (error) => console.error("There was an error", error))
-      .pipe(gulp.dest(`${svgPath}`))
+      .on("error", logError)
+      .pipe(dest(`${svgPath}`))
       .on("end", () => done())
   );
-});
+}
 
-gulp.task("rename-sprite", () => {
-  return gulp
-    .src(`${svgPath}/symbol/svg/sprite.symbol.svg`)
+function renameSprite() {
+  return src(`${svgPath}/symbol/svg/sprite.symbol.svg`)
     .pipe(rename(`${svgPath}/sprite.svg`))
-    .pipe(gulp.dest(`./`));
-});
+    .pipe(dest(`./`));
+}
 
-gulp.task("clean-sprite", () => del(`${svgPath}/symbol`));
+function cleanSprite() {
+  return del(`${svgPath}/symbol`);
+}
 
-gulp.task(
-  "svg-sprite",
-  gulp.series("build-sprite", "rename-sprite", "clean-sprite")
-);
+exports.buildSprite = series(
+  buildSprite,
+  renameSprite,
+  cleanSprite
+)
```

Comment:
None of these tasks are run on their own so I've combined them in `exports.buildSprite` and exported it.